ui-tests/src/test/java/io/jenkins/plugins/analysis/warnings/UiTest.java

```diff
@@ -251,6 +254,42 @@ protected void verifyCheckStyle(final Build build) {
                 .hasErrorMessages("Can't create fingerprints for some files:");
     }
 
+    protected void verifyPep8(final Build build) {
+        build.open();
+        AnalysisSummary pep8 = new AnalysisSummary(build, PEP8_ID);
+        assertThat(pep8).isDisplayed()
+                .hasTitleText(PEP8_NAME + ": 8 warnings")
+                .hasReferenceBuild(1)
+                .hasInfoType(InfoType.ERROR);
+
+        AnalysisResult pep8details = pep8.openOverallResult();
+        assertThat(pep8details).hasActiveTab(Tab.ISSUES)
```

Comment:
In den beiden Smoke Tests ist beim Öffnen der Details der Issues-Tab aktiv (und nicht der Categories-Tab). Daher musste ich auch beim GlobalConfigurationUiTest die Methode wieder überschreiben, damits wieder passt.

Ist das normal, dass an dieser Stelle der aktive Tab auf einmal anders ist? 
Weil jedes andere Plugin und der Groovy Parser selbst (wenn er das einzige Plugin ist) hier eben den Categories-Tab aktiv hat.